hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DistributedFileSystem.java

```diff
@@ -1107,6 +1107,7 @@ public Void next(final FileSystem fs, final Path p)
     }
 
     HdfsFileStatus[] partialListing = thisListing.getPartialListing();
+
```

Comment:
Please remove this before changing. Was just added to trigger hdfs tests by yetus.